lib/accounts.js

```diff
@@ -444,24 +444,10 @@ export class Accounts {
    * @param {Account} account - Account to update
    * @returns {Promise<Account>}
    * @tag Accounts
+   * @deprecated PUT call to accounts is deprecated, please use PATCH.
    */
   async update(account) {
-    checkString(account.accountID).or(Err.AccountID);
-
-    const token = await this.moov.getToken(account.accountID);
-
-    const result = await this.moov
-      .got({
-        url: `accounts/${account.accountID}`,
-        method: "PUT",
-        headers: {
-          authorization: `Bearer ${token.token}`,
-        },
-        json: account,
-      })
-      .json();
-
-    return result;
+    throw new Error('PUT call to accounts is deprecated, please use PATCH');
```

Comment:
Let's change the wording here as well.  And, we could have the update call make a call to patch().